### DISCUSSION TITLE
Add the ability to edit a Data Model from the content browser

### DISCUSSION THREAD

**Discussion by longjiang** [directus/directus#14305] [Feature Requests]
It would be really convenient if I could go directly from admin/content/TABLE_NAME to admin/settings/data_model/TABLE_NAME via a sidebar icon.
<img width="1101" alt="Screen Shot 2022-07-06 at 11 41 08 PM" src="https://user-images.githubusercontent.com/670049/177709436-361796a4-a8a6-457e-bbe3-cded812b93d3.png">

**Comment by azrikahar** [comment]
Currently we can already right click on a collection and navigate to it's data model page 👍

https://user-images.githubusercontent.com/42867097/177709973-b85d85db-ab8d-48e7-93dc-ad77677a7545.mp4

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
